app/src/main/java/com/example/paetz/yacguide/TourbookActivity.java

```diff
@@ -161,8 +171,9 @@ public void onClick(View v) {
                     startActivityForResult(intent, 0);
                 }
             };
+
             layout.addView(WidgetUtils.createCommonRowLayout(this,
-                    rock.getName() + " - " + route.getName(),
+                    WidgetUtils.getAscendSymbol(ascend.getStyleId()) + " " + rock.getName() + " - " + route.getName(),
```

Comment:
No symbol needed when we have a list for Säcke. --> Done in follow-up ticket #52.